src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -83,7 +88,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         private CSharpParseOptions(CSharpParseOptions other) : this(
-            languageVersion: other.LanguageVersion,
+            languageVersion: other.SpecifiedLanguageVersion,
```

Comment:
>languageVersion: other.SpecifiedLanguageVersion, [](start = 12, length = 48)

Would it make sense to simply copy effective LanguageVersion in this case rather than calling constructor that would have to recalculate it?